Core1Platform/BasicPhilosophy/04_ModuleOverview.md

```diff
@@ -0,0 +1,115 @@
+# Module overview
+
+In this part of the *Actindo Core1 Platform* documentation, you will find a brief overview on the most important *Actindo Core1 Platform* modules and technical components as well as a description of the data streams between these components.   
+See the interaction between the modules also in the overview graphic [Module overview &ndash; Diagram](./04a_GraphicModuleOverview.md).
+
+
+## Basic plugins and modules
+
+The following modules and plugins are the basis for the *Actindo Core1 Platform*. Their features control and process the data flow, the data processing, the data mapping as well as the communication with third party applications. In addition, it provides master data needed for a lot of modules.
```

Comment:
third-party (oben mit Bindestrich verwendet --> vereinheitlichen)

"a lot of" ist m.W. ein bisschen "informal" --> needed for many / different / a large number of modules
https://dictionary.cambridge.org/de/grammatik/britisch-grammatik/lots-a-lot-plenty

Reply:
done